wikimedia/wikipedia-ios
# Add support for gur(Frafra), guc (Wayuu), fat (Fanti), anp (Angika) languages

**Phabricator:** https://phabricator.wikimedia.org/T333596 and https://phabricator.wikimedia.org/T331550

### Notes
* Adding support for gur(Frafra), guc (Wayuu), fat (Fanti), anp (Angika) languages on the iOS app
* ~~Adding support for the Tachelhit script variant shi-tfng~~

### Test Steps
1. On settings > my languages, search for and add the new languages.
2. On search, try to see if you can search for articles in the newly added languages
